pkg/k8sutil/constants.go

```diff
@@ -20,7 +20,8 @@ const (
 	KubeConfigEnvVar = "KUBECONFIG"
 
 	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
-	// which is the namespace that the pod is currently running in.
+	// which is the namespace where the watch activity happens and where the pod is currently running in.
```

Comment:
I think we can strike "and where the pod is currently running in." now that we have a distinction between the watched namespace and the operator namespace.